### PR TITLE
GHA doc improvements

### PR DESCRIPTION
**Pull request by idelvall**
Address feedback from our session

### CODE REVIEW

File: docs/cloud/satellites/gha-runners.md [hunk 1]
```diff
@@ -17,22 +17,38 @@ These runners come with the Earthly CLI preinstalled and configured to use the s
...
+
+  ![token generate](./gha/token-generate.png)
+
+- Copy the token value to use it in the following step
+
+  ![token copy](./gha/token-copy.png)
 
 {% hint style='info' %}
-Follow the [official docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) for detailed information on how to create a GitHub token, and make sure to set an expiration long enough, since the integration won't work after the token expires.
+Alternatively, if you prefer creating a fine-grained tokens, make sure to set the following permissions for it: org: `organization_hooks:write`, `organization_self_hosted_runners:write`, repo: `repository_hooks:write`, `administration:write`
```
Comment (brandonSc) [inline docs/cloud/satellites/gha-runners.md:51]: Should we re-iterate here that this token will expire and they will need to come back and update it if they choose to use this method?

**Review by brandonSc — APPROVED** [review]

File: docs/cloud/satellites/gha-runners.md [hunk 2]
```diff
@@ -17,22 +17,38 @@ These runners come with the Earthly CLI preinstalled and configured to use the s
 
 Satellite-based GitHub Actions runners can be enabled for a particular repository or for all repositories of a GitHub organization at once.
 
-The integration process requires you to provide us with a GitHub token, so we can:
+The integration process requires you to be an organization/repository admin, and to provide us with a GitHub token, so we can:
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:20]:
```suggestion
The integration process requires you to be a GitHub organization or repository admin, and to provide us with a GitHub token, so we can:
```

File: docs/cloud/satellites/gha-runners.md [hunk 3]
```diff
@@ -17,22 +17,38 @@ These runners come with the Earthly CLI preinstalled and configured to use the s
...
 Follow the next steps to create such integrations:
 
 ### 1. Create a GitHub token
-Both GitHub classic and fine-grained tokens are supported, but depending on the type of installation (organization-wide or single-repository), the provided token requires different scopes:
 
-| Integration type | User type          | Classic token scopes          | Fine-grained token permissions                                       | 
-|------------------|--------------------|-------------------------------|----------------------------------------------------------------------|
-| Organization     | Organization admin | `admin:org_hook`, `admin:org` | `organization_hooks:write`, `organization_self_hosted_runners:write` |
-| Repository       | Repository admin   | `admin:repo_hook`, `repo`     | `repository_hooks:write`, `administration:write`                     |
+- Go to https://github.com/settings/tokens/new to create a new GitHub classic token.
```
Comment (vladaionescu) [inline docs/cloud/satellites/gha-runners.md:28]: We should avoid bare links

```suggestion
- Go to [GitHub's new token page](https://github.com/settings/tokens/new) to create a new GitHub classic token.
```

**Review by vladaionescu — COMMENTED** [review]
Minor remarks...

**Review by alexcb — APPROVED** [review]

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**